### PR TITLE
COMMERCE-12026 Chain of replacement SKUs doesn’t work for quick add to cart

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/COMMERCE-12026

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java [hunk 1]
```diff
@@ -265,6 +265,19 @@ public Sku toDTO(DTOConverterContext dtoConverterContext) throws Exception {
...
+			CPInstance replacedCPInstance =
+				_cpInstanceLocalService.fetchCPInstance(
+					cpInstance.getReplacementCProductId(),
+					cpInstance.getReplacementCPInstanceUuid());
+
+			return _checkReplacementCPInstance(replacedCPInstance);
+		}
+
+		return cpInstance;
+	}
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java:279]: nice!

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by saramart81** [comment]
ci:close

**Pull request closed without merging**